src/Controls/src/Core/Platform/Android/GenericGlobalLayoutListenerImproved.cs

```diff
@@ -0,0 +1,55 @@
+﻿#nullable enable
+using System;
+using Android.Views;
+using AView = Android.Views.View;
+using Object = Java.Lang.Object;
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	internal class GenericGlobalLayoutListenerImproved : Object, ViewTreeObserver.IOnGlobalLayoutListener
```

Comment:
For the backport I added this class to minimize the amount of code being changed. 

For .NET8 I've just modified `GenericGlobalLayoutListener` and updated everywhere that uses that code.